vispy/visuals/text/text.py

```diff
@@ -206,8 +206,10 @@ class TextVisual(Visual):
 
     Parameters
     ----------
-    text : str
-        Text to display.
+    text : str | list of str
+        Text to display. Can also be a list of strings.
+        Note: support for list of str will likely be removed soon
+        in favor of text collections.
```

Comment:
Why? This seems like a reasonable API to keep. 
In the long run, it is not at all clear how the collection and visual APIs will look when they are combined. I could imagine something like:

```
t1 = TextVisual("...", color1, font_size1)
t2 = TextVisual("...", color2, font_size2)
collection = TextCollection(t1 + t2)
```

.. but this API is quite cumbersome if you wanted to have, for example, 100k letters assigned to points in a scatter plot. That should look more like the API you have proposed here.